README.rst

```diff
@@ -292,6 +292,16 @@ the tag.
 For Git projects, the version relies on `git describe <https://git-scm.com/docs/git-describe>`_,
 so you will see an additional ``g`` prepended to the ``{revision hash}``.
 
+.. note::
+
+    According to `PEP 440 <https://peps.python.org/pep-0440/#local-version-identifiers>`_,
+    if a version includes a local component, the package cannot be published to public
+    package indexes like PyPI or TestPyPI. The disallowed version segments may
+    be seen in auto-publishing workflows or when a configuration mistake is made.
+
+    However, if you need to use local version identifiers in your packages, you
```

Comment:
I think that we shouldn't suggest people to use devpi just for the local version. This motivation is probably wrong. Instead, mention that some indexes, like devpi, allow those.